GameData/Kerbalism/Profiles/Default.cfg

```diff
@@ -203,8 +211,8 @@ Profile
     modifier = _WaterRecycler
     input = ElectricCharge@0.0446
     input = WasteWater@0.0001115
-    output = Water@0.0000892
-    output = Waste@0.0000223
+    output = Water@0.0001115
```

Comment:
This value is incorrect, you have not removed the waste from it

Reply:
WasteWater has higher density than Waste, you cannot just split the volume assuming they are equal density (check community resource pack for densities), the assumption here is that waste dissolved into the Water to form WasteWater and the volume of Water didn't decrease. What would be your proposal?

Reply:
A change in density will alter the mass not volume, if the water content of the wastewater is removed you are left with the waste content.
Imagine you have 80 Litres of water and you dissolve 20 Litres of waste into it, you are going to have 100 Litres of waste water regardless of the materials having differing densities.

Reply:
@PiezPiedPy You are talking about mixing liquids, but suspending minor amounts of solids inside a liquid doesn't necessarily behave that way (just imagine your urine). The density increase of WasteWater compared to Water means at least the mass per volume unit is increasing. The approximation that I took was that the additional mass per unit of WasteWater compared to Water i converted to volume units of Waste based on the density of Waste. An approximation I admit, but treating a solid inside a liquid as two liquids of same density, where they add up based on volume also seems wrong. Preservation of mass is a much stronger criteria than preservation of volume.